### ISSUE TITLE
clicSAND fails with run-time-error-429

### ISSUE THREAD

**Issue by ufechner7** [ClimateCompatibleGrowth/clicSAND#11]
I am following the online course https://www.open.edu/openlearncreate/course/view.php?id=11534

When trying to run my first model (exercise three) it fails with the run time error 429 activex-component-can-t-create-object.

The log file says:
```
------------------------------------------------------------------------------------------------------------------------------------------------------
Data file: C:\Users\ufechner\Documents\HO3\SAND_Interface_HO3.xlsm.txt
Model file: C:\Users\ufechner\Documents\HO3\OSeMOSYS_code.txt
GLPSOL Output file: C:\Users\ufechner\Documents\HO3\SAND_Interface_HO3.xlsm.lp
Results file: C:\Users\ufechner\Documents\HO3\SAND_Interface_HO3.xlsm.results.txt
Log file: C:\Users\ufechner\Documents\HO3\SAND_Interface_HO3.xlsm20240218184403.log.txt
------------------------------------------------------------------------------------------------------------------------------------------------------
Error extracting data:
Exception from HRESULT: 0x800A9C68
```

Any idea?

I am using Windows 10 and Microsoft office professional plus 2021.

**Comment by ufechner7** [comment]
Screenshot:
![Error](https://github.com/ClimateCompatibleGrowth/clicSAND/assets/4235229/b0bbe5d9-7a51-4d09-a108-87d30ac1d7e4)

**Comment by ufechner7** [comment]
I was able to fix the issue. When the error as shown above appeared, I clicked on "Debug". Then I saw the attached message.
![Error2](https://github.com/ClimateCompatibleGrowth/clicSAND/assets/4235229/d36e1b05-2e3e-4a32-a686-a75e686421e8)
I then changed the yellow line to look like this:
```
fn = Application.ThisWorkbook.Path & "\" & Application.ThisWorkbook.Name & ".txt"
```
Now it works.

I leave the issue open until someone creates a pull-request to close it.

See also: https://stackoverflow.com/questions/32000648/getting-object-variable-or-with-block-variable-not-set
